eosio_build.sh

```diff
@@ -35,6 +35,10 @@
 		printf "\\n\\tPlease cd into directory %s to run this script.\\n \\tExiting now.\\n\\n" "${CWD}"
 		exit 1
 	fi
+	if [ -f "${PWD}/CMakeCache.txt" ]; then
+		printf "\\n\\tPlease remove file %s/CMakeCache.txt before building EOS.IO.\\n \\tExiting now.\\n\\n" "${PWD}"
+		exit 1
+	fi
```

Comment:
I thought people used the eosio_build.sh script to rebuild their eosio.io after an update. This prevents eosio_build.sh from working for an existing checkout with a previous build.

Reply:
This file is required in directory /path/to/eos/build/
If this file is located in the eos/ directory CMake will write its build files to the eos/ directory and not the build directory which will cause:
cd build/
make make -j"${CPU_CORE}"
to error out i.e.
make: *** No targets specified and no makefile found.  Stop.

Reply:
Hmm. I build in source, but I guess build script does out of source. ok.